.gitignore

```diff
@@ -1,2 +1,6 @@
 yajsv
 build/
+
+# IntelliJ IDEA files
+.idea/
```

Comment:
I kind of lean the other way because not everyone has the proper ignores in their globals so better to protect each project.

That being said.. it isn't something I'm willing to lose sleep over so I've removed it in the latest commit.